website/docs/en/config/optimization.mdx

```diff
@@ -172,8 +172,8 @@ export default {
 };
 ```
 
-:::tip
-It is not recommended to disable this optimization as it may lead to a larger bundle size.
+:::danger
+It is danger to disable this optimization, as doing so may not only significantly increase the bundle size but also greatly slow down the build process.
```

Comment:
```suggestion
Disabling this optimization is dangerous, as it may significantly increase the bundle size and greatly slow down the build process.
```